src/test/Workflow_Examples.ml

```diff
@@ -82,13 +82,44 @@ let run_command_with_python_hack ~host cmd =
   let host = Host.parse host in
   Ketrew_client.submit (
     target (sprintf "Pyd: %S" cmd)
-      ~make:(daemonize ~using:`Python_daemon (Program.sh cmd) ~host)
+      ~make:(daemonize (Program.sh cmd)
+               ~using:`Python_daemon ~host
+               ~call_script:(fun script -> ["bash"; "--verbose"; script]))
   )
 (*M
 
 The `` `Python_daemon`` way of daemonizing was hacked together because
 MacOSX does not support the `nohup` and `setsid` commands any more.
 
+### Daemonize With “Apache Yarn”
+
+This function is like `run_command_with_lsf` but uses
+the
+[Yarn](http://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html)
+backend (Yarn comes from the Hadoop ecosystem and is used to request resources).
```

Comment:
I would make "Yarn comes from the Hadoop ecosystem and is used to request resources" into a separate sentence.  I'd also get rid of `the function yarn can...`